web/styles/zulip.css

```diff
@@ -2407,6 +2407,11 @@ body:not(.spectator-view) {
         border-radius: 4px;
         background-color: var(--color-background-image-loader);
         border: 1px solid var(--color-border-personal-menu-avatar);
+        overflow: hidden;
+        img {
+            /* Overcome vertical-align: middle */
+            vertical-align: top;
```

Comment:
This prevents the avatar (when not blocked) in the navbar from being lower by a pixel. (I could be remembering wrong but I feel like this lowering by a pixel sometimes doesn't happen, which is a little disturbing, but I haven't seen `vertical-align: top` cause a problem)

Reply:
Can you put a concise summary of this into the comment? Just saying we're overriding it helps a bit, but having enough information there for someone who wants to clean up that code to know what to test is really valuable.

Reply:
I think the better fix here is to set `display: block` on image. That corrects the droopiness from `vertical-align: middle`, but also better describes the behavior of an avatar image in this area.

Reply:
I changed it to `display: block` and it works as you said.

However I want to be sure I understood what you meant by "on `img` (and later the class that gets set here)". Are we set now?

Reply:
I was referring to the `.header-button-avatar-image` that you introduced in a later commit. Looks like that's all together now?

Also, you should just squash your `fixup` commit into the `navbar` one, it looks like.

Reply:
Yeah `.header-button-avatar-image` referred to this image. If you look at the code now, it's in place of `img` here.

Sure I can squash the commits. I just thought it'd be easier to review first before I do.

In general is it the common practice on Zulip to always force-push? Or just on small PRs like this?